doc/radosgw/group.rst

```diff
@@ -0,0 +1,100 @@
+===============
+Groups
+===============
+
+Ceph Object Gateway provides support for Amazon Groups. A group is a collection
```

Comment:
The Ceph Object Gateway (RGW) provides support for Amazon S3 _groups_.  A group is a collection of _users_. 

Below, what does `ARN` mean?